pyproject.toml

```diff
@@ -22,3 +22,6 @@ test = [
   "pytest-xdist",
 ]
 doc = ["sphinx"]
+
+[tool.black]
+line-length = 88
```

Comment:
Please don't do this. If this is for the hashes in the decorator being too long, I'd prefer we use `# fmt: off` and `# fmt: on` comments to avoid black's strictness around how the values should be formatted.

Reply:
Or, we can separately disable E501 in flake8, since formatting code via black basically makes that error redundant.

Reply:
Disabling `E501` seems good to me.